src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -561,6 +562,18 @@ class ArnoldRenderer : public IECoreScenePreview::Renderer
 				}
 				return;
 			}
+			else if( name == g_logFileNameOptionName )
+			{
+				if( value == NULL )
+				{
+					AiMsgSetLogFileName( "" );
```

Comment:
I'm assuming this means "keep logging to stderr or whereever it was going before"?